crates/biome_js_analyze/tests/specs/nursery/useReadonlyClassProperties/valid.ts.snap

```diff
@@ -0,0 +1,29 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: valid.ts
+---
+# Input
+```ts
+class ValidContainer {
+  // Public members might be modified externally
+  public publicMember: boolean;
+
+  // Protected members might be modified by child classes
+  protected protectedMember: number;
+
+  // This is modified later on by the class
+  private modifiedLater = 'unchanged';
+
+  public mutate() {
+    this.modifiedLater = 'mutated';
+  }
+
+  // This is modified later on by the class
+  #modifiedLaterPrivateField = 'unchanged';
+
+  public mutatePrivateField() {
+    this.#modifiedLaterPrivateField = 'mutated';
+  }
```

Comment:
```suggestion
```ts
class ValidContainer {
  // Public members might be modified externally
  public publicMember: boolean;

  // Protected members might be modified by child classes
  protected protectedMember: number;

  // // Holds the initial state and updated by the `mutate` method
  private modifiedLater = 'unchanged';

  public mutate() {
    this.modifiedLater = 'mutated';
  }

  // Holds the initial state and updated by the `mutatePrivateField` method
  #modifiedLaterPrivateField = 'unchanged';

  public mutatePrivateField() {
    this.#modifiedLaterPrivateField = 'mutated';
  }
```

imho: the comment `This is modified later on by the class`  could be improved with a more precise and clear description of the field's purpose and behavior. how do you think?